website/meta/universe.json

```diff
@@ -1,5 +1,27 @@
 {
     "resources": [
+	{
+	    "id": "numerizer",
+	    "title": "numerizer",
+	    "slogan": "Convert natural language numerics into ints and floats.",
+	    "description": "A SpaCy extension for Docs, Spans and Tokens that converts numerical words and quantitative named entities into numeric strings.",
+	    "github": "jaidevd/numerizer",
+	    "pip": "numerizer",
+	    "code_example": [
+		"from spacy import load",
+		"import numerizer",
+		"nlp = load('en_core_web_sm') # or any other model",
```

Comment:
While this reads "or any other model", the code at https://github.com/jaidevd/numerizer/blob/master/numerizer/numerizer.py has 
```
try:
    import spacy
    nlp = spacy.load('en_core_web_sm')
    SPACY_INSTALLED = True
except ImportError:
    SPACY_INSTALLED = False
```
which sort of implies a dependency on `en_core_web_sm`. I think that may be confusing for users?

```

Reply:
Ouch! It turns out that that line is unnecessary. I only need the `Doc`, `Token` and `Span` classes to exist - no need to have a pipeline there. I'll remove it and notify you. Thanks for pointing this out!